src/CppUTest/CommandLineTestRunner.cpp

```diff
@@ -145,8 +153,11 @@ bool CommandLineTestRunner::parseArguments(TestPlugin* plugin)
     return false;
   }
 
-  if (arguments_->isJUnitOutput())
+  if (arguments_->isJUnitOutput()) {
     output_= createJUnitOutput(arguments_->getPackageName());
+    if (arguments_->isVerbose())
+      output_ = createCompositeOutput(output_, createConsoleOutput());
+  }
   else
     output_ = createConsoleOutput();
   return true;
```

Comment:
In case of a Composite, the new/delete mismatch makes sense as the Composite must take ownership of it for it to be treated as one.

I don't like the changes needed for JUnitOutput as the functionality requested isn't related to jUnitOutput, but it is related to CommandLineRunner. So, to me, this change makes a lot more sense and JUnitOutput doesn't know about ConsoleTestOutput... as it shouldn't.